### PR TITLE
Feature univariate analysis

### PR DESCRIPTION
**Pull request by ty338jy**
Create functions for univariate analysis, features including:
- concatenate train set and test set
- create new columns based on existing columns (family size, age/fare interval, sex_Plcass)
- extract information from passenger names

### CODE REVIEW

File: Titanic Data Journey.ipynb
Comment (yuanjingshi) [inline Titanic Data Journey.ipynb:1]: I can't see the diff. This is the downside of changes in notebook. Do we need it? If not we can remove this change.

**Review by yuanjingshi — COMMENTED** [review]
Just a minor comment

File: environment.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 name: titanic_env
 channels:
   - conda-forge
+<<<<<<< HEAD
```
Comment (cameds58) [inline environment.yml]: The change is not merged

**Review by ps2016-mt — APPROVED** [review]
looks good

**Review by jps215 — APPROVED** [review]
Looks good 👍

**Pull request merged**